azure-fluid-relay-preview-pr/articles/howtos/hosting.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Hosting Fluid Example
+description: Detailed explanation about how fluid examples can be hosted on Azure App Service and Static Web App
+---
+
+This article demonstrates how to deploy your Fluid examples to Azure App Service and Static Web Apps. [FluidHelloWorld](https://github.com/microsoft/FluidHelloWorld) repository is a Fluid example which contains a simple app that enables all connected clients to roll a dice and view the result. `FluidHelloWorld` is framework agnostic and provides sample code for rendering three different views: React, Vue and VanillaJS. Please pick the one that suits your development requirements.
+
+# Pre-requisites
+1. Add the following extensions to Visual Studio Code: 
+    - [Azure Account](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account): It is required to sign in to your `Azure Subscription`.
+    - [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice): It is required for deploying app using `App Services`.
+    - [Azure Static Web App](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps): It is required for deploying app using `Static Web Apps`.
+
+These extensions allow the developer to use Azure directly from Visual Studio Code.
+
+2. Run the following command to clone the repository:
+
+```shell
+git clone https://github.com/microsoft/FluidHelloWorld
+```
+
+# Connecting to Azure Fluid Relay Service
+You can connect to `Azure Fluid Relay Service` by providing the tenant ID and key that is uniquely generated for you when onboarding to the service. The `tokenProvider` can be generated using any one of the following methods: 
+
+1. `InsecureTokenProvider`: This type of implementation is only recommended for development purposes as it exposes the tenant key secret in your client-side code bundle.
```

Comment:
Let's perhaps include some guidance on what file this code would need to get edited to switch between the two (i.e. where is the config we pass to the AzureClient located)